doc/nrf/ug_partition_manager.rst

```diff
@@ -0,0 +1,203 @@
+
+      after: list
+         Place the partition after the first existing partition in the list.
+
+   The valid values in the lists are `app`, `end`, or the name of any partition.
+   It is not possible to place the partition after `end`.
+
+sub_partitions: list
+   Define sub partitions which span across one or more other partitions.
+   The sub partitions split the total size of the spanned partitions, and therefore have equal size.
+   This property cannot be used together with the `placement` property.
+   When this property is set for a partition, it is required that the `span` property is also set.
+
+span: list
+   When creating sub_partitions, this option lists what partitions the sub_partitions should span across.
+   This property cannot be used together with the `placement` property.
+
+.. _pm_yaml_preprocessing:
+
+Configuration file preprocessing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each :file:`pm.yml` file is preprocessed.
+Example of preprocessing is shown below:
+
+.. code-block:: yaml
+
+   #include <autoconf.h>
+
+   # 'b0' is the name of the image partition.
+   b0:
+
+     # b0 is placed before the mcuboot partition if the mcuboot partition
+     # exists, otherwise it is stored before the app partition.
+     placement: {before: [mcuboot, app]}
+
+   # Don't define the provision partition if the SoC is nRF9160, this because
+   # the provisioning data will be stored in the UICR->OTP data region.
+
+   #ifndef CONFIG_SOC_NRF9160
+
+   # 'provision' is the name of the placeholder partition.
+   provision:
+     # This partition is stored at the very end of flash.
+     placement: last
+
+   #endif /* CONFIG_SOC_NRF9160 */
+
+.. _pm_yaml_partition_types:
+
+Partition types
+~~~~~~~~~~~~~~~
+
+It is required that each :file:`pm.yml` defines exactly one *image partition*.
+This is done by using the same name for the partition as the image name.
+
+All other partitions which have the `placement` property set are *placeholder
+partitions*.
+
+Partitions which have the `sub_partitions` property set are *phony partitions*,
+and do not occupy space in flash.
+
+In addition to the configuration provided in :file:`pm.yml`, each partition with the `placement` property set must have a corresponding size configuration in the image's ``Kconfig`` file.
+The format of this configuration is as follows:
+
+.. code-block:: none
+
+   config PM_PARTITION_SIZE_[PARTITION_NAME]
+   hex "Flash space reserved for [partition name]."
+   default 0xD000
+   help
+     Flash space set aside for [partition name]. Note, the name
+     of this configuration needs to match the requirements set by the
+     script 'partition_manager.py'. See pm.yaml.
+
+.. _pm_build_system:
+
+Build system
+============
+This section describes how the Partition Manager is included by the Zephyr build system.
+
+If one or more sub-images are included in a build, a set of properties for that sub-image is appended to a global list.
```

Comment:
What are the actual properties? I see the list of what the properties contain, but not the actual CMake property names, so I don't know how to grep the source code to find out more about them.

Reply:
Next revision will have all properties, will ensure that this is described in the docs.